src/main/java/com/revature/domain/Eval.java

```diff
@@ -3,10 +3,10 @@
 import java.io.Serializable;
 import java.sql.Date;
 import java.util.Objects;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Objects'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)